bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Path.java

```diff
@@ -201,10 +211,10 @@ public Path (Device device, PathData data) {
 public void addArc (float x, float y, float width, float height, float startAngle, float arcAngle) {
 	if (width == 0 || height == 0 || arcAngle == 0) return;
 	Drawable drawable = getDevice();
-	x = DPIUtil.autoScaleUp(drawable, x);
-	y = DPIUtil.autoScaleUp(drawable, y);
-	width = DPIUtil.autoScaleUp(drawable, width);
-	height = DPIUtil.autoScaleUp(drawable, height);
+	x = DPIUtil.autoScaleUp(drawable, x, initialZoom);
```

Comment:
Create a different PR to rename `autoScaleUp` --> `scaleUp`

Reply:
We will do that for the whole project after we have merged everything :)